.discourse-compatibility

```diff
@@ -1,2 +1,2 @@
+< 3.2.0.beta4-dev: b3c727286bdad3661d6cb2b83e369588e91b458f
```

Comment:
I ran into compatibility problems when using an unreleased version previously because `3.2.0.beta4-dev` can represent a wide range of Discourse core commits. Does the feature being introduced here rely on a Discourse core change that has yet to be released? If so, we might need to be more careful here.

Reply:
The discovery service was included in the 3.2.0.beta3 release, so we should be safe 👌